bld/test/compat/.nuget/NuGet.Config

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="true" />
+  </solution>
+  <config>
+    <add key="repositorypath" value="..\packages" />
+  </config>
+    <packageSources>
+        <add key="nugetv2" value="https://www.nuget.org/api/v2/" />
+        <add key="nugetv2dotnet" value="https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/" />
+        <add key="nugetv3" value="https://api.nuget.org/v3/index.json" />
+        <!-- To enable LocalFeed for testing uncomment the following line -->
+        <add key="Local" value=".\LocalPackages" />
```

Comment:
Yep, on purpose.  But the comment is weird.  I copied this from connected factory.  Will update.